test/test_ops.py

```diff
@@ -463,6 +439,115 @@ def _fn(*sample_args, **sample_kwargs):
                 graph = traced.graph_for(*inp)
                 FileCheck().check(op_name).check_not(variant_name).run(graph)
 
+    # Validates ops implement the correct out= behavior
+    # See https://github.com/pytorch/pytorch/wiki/Developer-FAQ#how-does-out-work-in-pytorch
+    #   for a description of the correct behavior
+    @ops(op_db, dtypes=OpDTypes.supported, allowed_dtypes=(torch.float,))
+    def test_out(self, device, dtype, op):
```

Comment:
Note: (from offline review with @ngimel) update to handle ops which produce iterables of Tensors as well